src/index.js

```diff
@@ -225,14 +225,15 @@ export default class DRSS {
      * @returns {HTMLStyleElement} the style element (id #drssHead)
      */
     static _getStyleElement() {
-        if (!this._initialized) return false;
```

Comment:
I didn't think it made sense for this to return false. I think it returning null makes more sense, but if we're gonna return null, I think its probably better to just return nothing at all (so undefined in this case)

Reply:
Cool

Reply:
Seems pretty good